modules/ai/assets/js/editor/pages/form-layout/components/prompt-form.js

```diff
@@ -63,12 +66,25 @@ const GenerateButton = ( props ) => (
 	</GenerateSubmit>
 );
 
-const PromptForm = forwardRef( ( { isActive, isLoading, showActions = false, onSubmit, onBack, onEdit }, ref ) => {
+const PromptForm = forwardRef( ( {
+	attachments,
+	isActive,
+	isLoading,
+	showActions = false,
+	onSubmit,
+	onBack,
+	onEdit,
+}, ref ) => {
 	const [ prompt, setPrompt ] = useState( '' );
 	const { isEnhancing, enhance } = usePromptEnhancer( prompt, 'layout' );
 	const previousPrompt = useRef( '' );
+	const { attachmentsTypes } = useConfig();
 
-	const isInteractionsDisabled = isEnhancing || isLoading || ! isActive || '' === prompt;
+	const isInteractionsDisabled = isEnhancing || isLoading || ! isActive || ( '' === prompt && attachments.length < 1 );
```

Comment:
```suggestion
	const isInteractionsDisabled = isEnhancing || isLoading || ! isActive || ( ! prompt && ! attachments.length );
```

maybe?
not sure if it's more readable though, WDYT?